src/components/UserInfo.js

```diff
@@ -11,7 +12,13 @@ export default class UserInfo {
   }
 
   setUserInfo(inputValues) {
-    this._profileTitle.textContent = inputValues.title;
+    this._profileTitle.textContent = inputValues.name;
     this._profileDescription.textContent = inputValues.about;
+    // console.log(this._avatarElement);
+  }
+
+  setUserAvatar(inputValues) {
```

Comment:
It's great to have a method for setting the avatar, because it's controlled by a different form